lua/autorun/server/sv_daily_restart.lua

```diff
@@ -445,8 +445,8 @@ end
 
 -- Calculates up to 23:59:59 to wait until restart
 local function waitUntilRestartHour()
-    local currentMinute = tonumber( os.date( "%M" ) )
-    local currentSecond = tonumber( os.date( "%S" ) )
+    local currentMinute = tonumber( os.date( "!%M" ) )
+    local currentSecond = tonumber( os.date( "!%S" ) )
```

Comment:
```suggestion
    local timeTbl = os.date( "!*t" )
    local currentMinute = timeTbl.min
    local currentSecond =  timeTbl.sec
```

Reply:
`*t` causes it to generate all time formats, which would be less performant. I can combine the minute and seconds together in one string, however.

Reply:
yeah but it doesnt have to do the string to int conversion

Reply:
Please dont split it thats worse than both options